introduction/edge-functions.mdx

```diff
@@ -0,0 +1,59 @@
+# Edge functions
+Edge Functions are server-side functions that run directly within your database environment. Edge functions in SQLite Cloud ensure maximum performance and minimal latency by running functions on the same node as your database.
+
+You can write edge functions directly in the SQLite Cloud dashboard using JavaScript, TypeScript, SQL, or Python (experimental). You can [import modules](https://bun.sh/docs/runtime/autoimport) into your JavaScript and TypeScript edge functions. Module imports for Python are not currently supported.
+
+Edge functions can be called remotely over HTTP or Websockets via API, or triggered by database events via SQLite Cloud [Webhooks](https://docs.sqlitecloud.io/docs/introduction/webhooks). Each function runs in an isolated environment using the Bun runtime. 
+
+## Getting Started
+1. Navigate to the Edge Functions page from your dashboard. [add image]
+1. Under "Development", select "Edge Function", then click the "Create" button. [add image]
+1. Write and test your function.
+    1. Select the database you want to access and an api key if necessary
+    2. When you're finished, click deploy. [image - hello world function]
+
+#### Note:
+1. Functions must return a JSON-serializable object
```

Comment:
@Gioee @TizianoT @gionatamettifogo Please add any other hard-requirements for using our edge functions

Reply:
Returned JSON as to be formated as follow:
```
return {
    data: {
        result: 1,
        message: "hello world",
    }
}
```

Reply:
This if working in JS or TS.
Working in SQL the JSON is created by the gateway.